shell/browser/ui/win/electron_desktop_window_tree_host_win.cc

```diff
@@ -37,6 +38,15 @@ bool ElectronDesktopWindowTreeHostWin::PreHandleMSG(UINT message,
   return native_window_view_->PreHandleMSG(message, w_param, l_param, result);
 }
 
+void ElectronDesktopWindowTreeHostWin::PostHandleMSG(UINT message,
+                                                     WPARAM w_param,
+                                                     LPARAM l_param) {
+  // See ui/views/win/hwnd_message_handler.cc:3213 for why
+  // this is WM_RBUTTONUP and not WM_NCRBUTTONUP.
+  if (message == WM_RBUTTONUP)
+    electron::api::WebContents::SetDisableDraggableRegions(false);
```

Comment:
To align with linux https://github.com/electron/electron/pull/45848/files, this could be moved closer to where we handle sytem-context-menu on windows https://github.com/electron/electron/blob/21ad7cdda59f8cd451384fb16bac3aeb19095c17/shell/browser/native_window_views_win.cc#L291-L299